tsconfig.json

```diff
@@ -0,0 +1,17 @@
+{
+  "compilerOptions": {
+    "target": "ESNext",
+    "module": "ESNext",
+    "moduleResolution": "node",
+    "strict": true,
+    "jsx": "preserve",
```

Comment:
please check carefully every option you're added, it should have particular meaning for this project